src/main/java/uk/ac/ebi/tsc/portal/api/team/service/TeamService.java

```diff
@@ -838,11 +838,14 @@ private static String getCloudProviderPathFromDeploymentApplication(DeploymentAp
 		return null;
 	}
 
-	public TeamResource setManagerUserNames(TeamResource teamResource, String token) {
+	public TeamResource setManagerUsernamesAndEmails(TeamResource teamResource, String token) {
 		List<String> managerUserNames = new ArrayList<>();
+		List<String> managerEmails = new ArrayList<>();
 		try {
 			managerUserNames = this.domainService.getAllManagersFromDomain(teamResource.getDomainReference(), token)
 					.parallelStream().map(manager -> manager.getUserReference()).collect(Collectors.toList());
+			managerEmails = this.domainService.getAllManagersFromDomain(teamResource.getDomainReference(), token)
+					.parallelStream().map(manager -> manager.getEmail()).collect(Collectors.toList());
```

Comment:
And as previously you are calling the same method from domainService twice!! Isn't the first call returning the value already?

Reply:
Was an error oversight replaced it.